vmdb/app/models/ems_refresh/parsers/kubernetes.rb

```diff
@@ -0,0 +1,140 @@
+module EmsRefresh::Parsers
+  class Kubernetes
```

Comment:
Can this derived from the Parser Base?  Then you get process_collection for free (I think?) and we can keep that consistent as well.  It you don't get process_collection, maybe we should copy the one from openstack or just properly refactor it here.  @blomquisg